pkg/container-disk/container-disk_test.go

```diff
@@ -318,11 +348,15 @@ var _ = Describe("ContainerDisk", func() {
 					f, err = os.Create(fmt.Sprintf("%s/pods/%s/volumes/kubernetes.io~empty-dir/container-disks/disk_1.sock", tmpDir, "poduid"))
 					Expect(err).ToNot(HaveOccurred())
 					Expect(f.Close()).To(Succeed())
-					vmi = api.NewMinimalVMI("fake-vmi")
+
+					By("Creating a new VMI with multiple container disks")
+					vmi = libvmi.New(
+						libvmi.WithContainerDisk("r0", someImage),
+						libvmi.WithContainerDisk("r1", someImage),
+						libvmi.WithContainerDisk("r2", someImage),
+					)
+					vmi.UID = "6789"
```

Comment:
Where does this come from?

Reply:
I Create a new VMI with multiple container disks
instead of
```go
vmi = api.NewMinimalVMI("fake-vmi")
appendContainerDisk(vmi, "r0")
appendContainerDisk(vmi, "r1")
appendContainerDisk(vmi, "r2")
```
I used  libvmi
```go
vmi = libvmi.New(
libvmi.WithContainerDisk("r0", someImage),
libvmi.WithContainerDisk("r1", someImage),
libvmi.WithContainerDisk("r2", someImage),
)
```
I needed to provide to the WithContainerDisk image name 
```go
const someImage = "someimage:v1.2.3.4"
```
also I will remove the ``` vmi.UID = "6789" ```